### PR TITLE
[ENG-724] Provide build information for Python

### PR DESCRIPTION
**Pull request by tebeka**
Still need work in the extension, @RonenMars here for that :)

### CODE REVIEW

File: runtimes/pythonrt/py.go [hunk 1]
```diff
@@ -39,6 +40,44 @@ func createTar(fs fs.FS) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+// Copy fs to file so Python can inspect
+// TODO: Once os.CopyFS makes it out we can remove this
```
Comment (daabr) [inline runtimes/pythonrt/py.go:44]: Can you add a reference link for that?

File: runtimes/pythonrt/pythonrt.go [hunk 2]
```diff
@@ -140,12 +140,32 @@ func (py *pySvc) Build(ctx context.Context, fs fs.FS, path string, values []sdkt
 		return sdktypes.InvalidBuildArtifact, err
 	}
 
+	exports, err := pyExports(ctx, py.pyExe, fs)
+	if err != nil {
+		py.log.Error("get exports", zap.Error(err))
+		return sdktypes.InvalidBuildArtifact, err
+	}
+
+	buildExports := kittehs.Transform(exports, func(e Export) sdktypes.BuildExport {
```
Comment (daabr) [inline runtimes/pythonrt/pythonrt.go]: Don't learn from Itay. Long anonymous functions and 3 levels of indentation are not cool. :)

It's up to you if you want to keep this as-is, but please be thoughtful about this in the future.

**Review by daabr — APPROVED** [review]
Approved. But the files in this PR are severely lacking comments. Please dedicate some time for this soon - the more you wait, the harder it will be to do it.

**Review by ig-ra — APPROVED** [review]

File: runtimes/pythonrt/py.go (hunk 1 above)
Comment (tebeka) [reply runtimes/pythonrt/py.go:44]: Done.

File: runtimes/pythonrt/pythonrt.go (hunk 2 above)
Comment (tebeka) [reply runtimes/pythonrt/pythonrt.go]: Moved to external function.

**Pull request merged**